homework12/README.md

```diff
@@ -0,0 +1,38 @@
+Task:
+
+Using ORM framework of your choice, create models classes created in Homework 6 (Teachers, Students, Homework and others). - Target database should be sqlite (filename main.db localted in current directory) - ORM framework should support migrations.
+
+Utilizing that framework capabilities, create
+a migration file, creating all necessary database structures.
+a migration file (separate) creating at least one record in each created database table
+(*) optional task: write standalone script (get_report.py) that retrieves and stores the following information into CSV file report.csv
+for all done (completed) homeworks:
+Student name (who completed homework) Creation date Teacher name who created homework
+Utilize ORM capabilities as much as possible, avoiding executing raw SQL queries.
+
+To run this project:
+
+1.Copy link to this repo `git clone https://github.com/IuliiaLanchuk/hometasksEpam.git develop`
+
+2.Install python and then in command line add python and pip to PATH variables:
+`setx PATH "C:\Users\User_name\AppData\Local\Programs\Python\Python39\;%PATH%"` and
+`setx PATH "C:\Users\User_name\AppData\Local\Programs\Python\Python39\Scripts\;%PATH%"`
+
+3.Set up used libs in this project: `pip install -r requirements.txt`
```

Comment:
This still does not work for me.  Have you tried to run this command?

Reply:
Oh, my bad, here is `pip install -r requirements-dev.txt`. I am so inattentive:)

Reply:
Actually, the idea of `dev` suffix is to split requirements for development (linters, formatters and pytest) and requirements for running the project.   I usually have 2 files in the repo `requirements-dev.txt` and `requirements.txt`

Reply:
Aaa, okey. But in that project is it would be enough to have only one requirements-dev.txt? If we split them into 2 files , in requirements.txt there would be only one line - Django==3.2

Reply:
For that case just leave it as is.